static/app/components/quickTrace/index.tsx

```diff
@@ -86,8 +87,38 @@ export default function QuickTrace({
   const traceLength = quickTrace.trace && quickTrace.trace.length;
   const {root, ancestors, parent, children, descendants, current} = parsedQuickTrace;
 
+  const replayId = event.tags?.find(({key}) => key === 'replayId')?.value;
+
   const nodes: React.ReactNode[] = [];
 
+  if (organization.features.includes('session-replay-ui')) {
+    nodes.push(
+      <Projects
+        key="replay-projects"
+        orgId={organization.slug}
+        slugs={[current.project_slug]}
+      >
+        {({projects}) => {
+          const project = projects.find(p => p.slug === current.project_slug);
+          if (projectSupportsReplay(project)) {
+            return (
+              <EventNodeReplay
+                data-test-id="replay-node"
+                to={`/organizations/${organization.slug}/replays/${current.project_slug}:${replayId}`}
```

Comment:
this url will be broken if there is no `replayId`

The button we're replacing was linking to `#breadcrumbs`, lets keep that the same, if there's a replayId